### DISCUSSION TITLE
amountPaid on the Greenfield Invoice object

### DISCUSSION THREAD

**Discussion by AryanJ-NYC** [btcpayserver/btcpayserver#2525] [Ideas / Feature Requests]
# User Story
As a merchant, I want to know how much a buyer actually paid (rather than only the amount of the invoice).

# Background
Currently, the `Invoice` object returned by [GetInvoice](https://docs.btcpayserver.org/API/Greenfield/v1/#operation/Invoices_GetInvoice) only returns the `amount` of the invoice, rather than the `amountPaid` by the user.

# Acceptance Criteria
`amountPaid` is on the `invoice` object.

**Comment by NicolasDorier** [comment]
https://github.com/btcpayserver/btcpayserver/pull/6747
